control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/assets.go

```diff
@@ -0,0 +1,1568 @@
+package kubevirt_assets
+
+import (
+	"bytes"
+	"embed"
+	"io"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+
+	"github.com/hoisie/mustache"
```

Comment:
I think it would be better to use the golang standard https://pkg.go.dev/text/template instead. The primary reasoning is that we would prefer not to pull in new dependencies when golang already gives us a tool like this.